services/gitea/gitea-languages-count.tester.js

```diff
@@ -3,24 +3,29 @@ import { createServiceTester } from '../tester.js'
 
 export const t = await createServiceTester()
 
-t.create('language count (empty repo)')
+t.create('language count').get('/gitea/tea.json').expectBadge({
+  label: 'languages',
+  message: Joi.number().integer().positive(),
+})
+
+t.create('language count (empty repo) (self-managed)')
   .get(
-    '/CanisHelix/shields-badge-test-empty.json?gitea_url=https://codeberg.org',
+    '/CanisHelix/shields-badge-test-empty.json?gitea_url=https://try.gitea.io',
```

Comment:
https://try.gitea.io says
"THIS IS A TEST INSTANCE ONLY! REPOSITORIES CAN BE DELETED AT ANY TIME!"
at the top of it in massive letters.

Having integration tests that run against this says to me that at some point these tests will unexpectedly break due to an externality. I'd be inclined to stick with the custom instance tests running against codeberg.